src/keri/db/dbing.py

```diff
@@ -137,17 +137,20 @@ def splitKey(key, sep=b'.'):
     return tuple(splits)
 
 
-def splitKeyON(key):
+def splitKeyON(key, *, sep=b'.'):
     """
     Returns list of pre and int on from key
     Accepts either bytes or str key
     ordinal number  appears in key in hex format
     """
     if isinstance(key, memoryview):
         key = bytes(key)
-    pre, on = splitKey(key)
+    top, on = splitKey(key, sep=sep)
     on = int(on, 16)
-    return (pre, on)
+    return (top, on)
+
+splitSnKey = splitKeyON # alias so intent is clear, sn vs fn; backport of 1.2.x alias
+splitFnKey = splitKeyON # alias so intent is clear, sn vs fn; backport of 1.2.x alias
 
 splitKeySN = splitKeyON  # alias so intent is clear, sn vs fn
 splitKeyFN = splitKeyON  # alias so intent is clear, sn vs fn
```

Comment:
Why are these seemingly repeated?  Also why are they global?

Reply:
It is repeated because I ported the two names from the 1.2.x release version (now `main`). Those two are global to the `keri.db.dbing` module because that's how Sam wrote them.